aten/src/ATen/core/type.cpp

```diff
@@ -175,7 +175,7 @@ TypePtr attemptToRecoverType(const IValue& ivalue) {
     if (ivalue_list.size() == 0) {
       return ListType::create(VarType::create("t"));
     }
-    return ListType::create(attemptToRecoverType(ivalue_list[0]));
+    return ListType::create(attemptToRecoverType(ivalue_list.get(0)));
```

Comment:
My constraint is that this cannot regress the API of what we had before. Why can't operator[] work?

Reply:
I changed almost all of them back to `[]`.